### PR TITLE
Fixed compression formats

### PR DESCRIPTION
**Pull request by gluka-CBS**
Fixed compression formats and password protected zip

### CODE REVIEW

**Comment by ader1990** [comment]
Hello, can you explain what your fix does?
Also, please remove any tabs from the indentation to keep consistency.

File: Config.psm1 [hunk 1]
```diff
@@ -54,10 +54,11 @@ function Get-AvailableConfigOptions {
         @{"Name" = "install_maas_hooks"; "DefaultValue" = $false; "AsBoolean" = $true;
           "Description" = "If set to true, MAAS Windows curtin hooks will be copied to the image root directory."},
          @{"Name" = "compression_format";
-          "Description" = "Select between tar, gz, zip formats or any combination between these."},
+          "Description" = "Select between tar, gz, tar.gz or zip formats"},
```
Comment (ader1990) [inline Config.psm1]: you can choose tar.zip and it will work

File: Config.psm1 [hunk 2]
```diff
@@ -54,10 +54,11 @@ function Get-AvailableConfigOptions {
...
         @{"Name" = "zip_password";
           "Description" = "If this parameter is set, after the image is generated,
-                           a password protected zip archive with the image will be created."},
+                           a password protected zip archive with the image will be created. Previous parameter, compression_format,
```
Comment (ader1990) [inline Config.psm1]: line too long, keep the line length to be at least reviewable without horizontal scrolling in github.

File: Config.psm1 (hunk 1 above)
Comment (gluka-CBS) [reply Config.psm1]: Hello,

It fixes the cases where 
virtual_disk_format=RAW
image_type=MAAS
are selected along with different compression formats.
I will check the identation

**Comment by ader1990** [comment]
I do not know that this patch does the right thing, I have revise all the workflow, document and see the missing links. I am sure this patch does not solve all the issues that are there and it probably makes the workflow more complicated or incomplete.

**Pull request closed without merging**